WordPress/src/main/java/org/wordpress/android/ui/main/MySiteFragment.java

```diff
@@ -1114,11 +1112,7 @@ public void onEventMainThread(StorySaveEvents.StorySaveResult event) {
                                         event.getStoryIndex()) + ""
                         );
 
-                        // TODO WPSTORIES add TRACKS: the putExtra described here below for NOTIFICATION_TYPE
-                        // is meant to be used for tracking purposes. Use it!
-                        // TODO add NotificationType.MEDIA_SAVE_ERROR param later when integrating with WPAndroid
-                        //        val notificationType = NotificationType.MEDIA_SAVE_ERROR
-                        //        notificationIntent.putExtra(ARG_NOTIFICATION_TYPE, notificationType)
+                        intent.putExtra(ARG_NOTIFICATION_TYPE, NotificationType.STORY_SAVE_ERROR);
```

Comment:
minor note: this is related to the work done on #12068, but put it in this PR as I'm searching for all `// TODO WPSTORIES add TRACKS` comments and doing changes accordingly :)